deploy/AzureAppServices/content/install.cmd

```diff
@@ -0,0 +1,18 @@
+
+REM Create logging directory for tracer version
+mkdir D:\home\LogFiles\Datadog\v1_10_39
```

Comment:
Wouldn't having a log directory name that doesn't reflect the Tracer version be better? I don't see why you would have separate sub-directories for the logs from each version. Or maybe I'm not understanding.

Reply:
It was easier for me, releasing so many versions in a row, for differentiating between them. 
We could go either way I am not stuck on it.

Reply:
Makes sense for your iterating.